WireUI/Sources/WireSidebar/Views/SidebarViewController.swift

```diff
@@ -19,94 +19,113 @@
 import SwiftUI
 import WireFoundation
 
-public final class SidebarViewController: UIHostingController<SidebarViewAdapter<AnyView>> {
```

Comment:
Any specific reason to change to a regular `UIViewController` subclass? I've nothing against it, just wondering why the change?

Reply:
Yes, because I need the view controller to be public, however, I prefer not to expose the type `SidebarViewAdapter`.  Thats' why I chose to add the hosting controller as a child instead.